biscuit-parser/src/parser.rs

```diff
@@ -452,8 +452,8 @@ fn binary_op_2(i: &str) -> IResult<&str, builder::Binary, Error> {
         value(Binary::GreaterOrEqual, tag(">=")),
         value(Binary::LessThan, tag("<")),
         value(Binary::GreaterThan, tag(">")),
-        value(Binary::Equal, tag("==")),
-        value(Binary::NotEqual, tag("!=")),
```

Comment:
imo Equal and NotEqual could continue to be parsed with `===` and `!==`, in line with other languages strict equality tests.

Reply:
Ok I'll add that